test/test_auth.py

```diff
@@ -0,0 +1,99 @@
+import os
+import requests
+
+
+def login(webapi, cookies=None, csrf=False, profile=None):
+
+    data = {"password": "Yoloswag"}
+    if profile:
+        data["profile"] = profile
+
+    return requests.post(webapi + "/login",
+                         cookies=cookies,
+                         data=data,
+                         headers=None if csrf else {"X-Requested-With": ""})
+
+
+def test_request_no_auth_needed(monkeypatch, tmp_path, moulinette_webapi):
```

Comment:
`monkeypatch` and `tmp_path` not used:
```suggestion
def test_request_no_auth_needed(moulinette_webapi):
```